lib/autoload/kuroko2/command/executor.rb

```diff
@@ -19,7 +19,8 @@ def initialize
                    elsif worker_id == (Command::Executor.num_workers - 1)
                      Command::Monitor.new(hostname: @hostname, worker_id: worker_id)
                    else
-                     @worker = Worker.where(hostname: @hostname, worker_id: worker_id, queue: @queue).first_or_create!
+                     @worker = Worker.where(hostname: @hostname, worker_id: worker_id, queue: @queue).first_or_initialize!
+                     @worker.update_column(:suspendable, true)
```

Comment:
`update_column` skips varidations and callbacks and doesn't update updated_at. Why did you choose update_column instead of `update!` ?
http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update_column